crates/witness-generator/src/lib.rs

```diff
@@ -238,6 +238,15 @@ impl StatelessValidationFixture {
         })?;
         Self::from_json(&contents)
     }
+    
+    /// Creates a new fixture from stateless input and a name.
```

Comment:
```suggestion
    /// Creates a new valid fixture from stateless input and a name.
```

Since we still have some exceptional inputs when testing (with `success = false`), this method might be a bit confusing because it assumes it's a valid input (sorry to be nitpicking).